google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -206,7 +209,10 @@ static BigtableDataClient createWithClientContext(
    * }</pre>
    *
    * @throws com.google.api.gax.rpc.ApiException when a serverside error occurs
+   * @deprecated Please use {@link BigtableDataClient#exists(TargetId, String)} instead. Create a
+   *     TargetId for a table with {@link TableId#of(String)}.
```

Comment:
here and everywhere else what do you mean by `Create a TargetId for a table with {@link TableId#of(String)}`?  I think a pointer to the new TargetId method should be enought